component/src/test/java/com/celements/structEditor/OptionTagPageTypeTest.java

```diff
@@ -0,0 +1,185 @@
+    DocumentReference docRef2 = new DocumentReference("wikiName", "Celements", "gfrt");
+
+    expect(strucDataEdSrvMock.getSelectCellDocRef(eq(docRef))).andReturn(
+        Optional.<DocumentReference>of(docRef2));
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_VALUE))).andReturn(Optional.<String>of("Test"));
+    expect(strucDataEdSrvMock.getCellValueAsString(eq(docRef2), eq(
+        getContext().getDoc()))).andReturn(Optional.<String>of("Test"));
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_DISABLED))).andReturn(Optional.of(false));
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_VALUE))).andReturn(Optional.<String>absent());
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_LABEL))).andReturn(Optional.<String>absent());
+    replayDefault();
+    optionTagPageType.collectAttributes(attrBuilder, docRef);
+    verifyDefault();
+    List<CellAttribute> cellAttrList = attrBuilder.build();
+    assertEquals(1, cellAttrList.size());
+    assertEquals("selected", cellAttrList.get(0).getName());
+  }
+
+  @Test
+  public void test_collectAttributes_field_disabled() throws Exception {
+
+    DocumentReference docRef = new DocumentReference("wikiName", "Celements", "TestXClassName");
+
+    expect(strucDataEdSrvMock.getSelectCellDocRef(docRef)).andReturn(
+        Optional.<DocumentReference>absent());
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_DISABLED))).andReturn(Optional.of(true));
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_VALUE))).andReturn(Optional.<String>absent());
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_LABEL))).andReturn(Optional.<String>absent());
+
+    replayDefault();
+    optionTagPageType.collectAttributes(attrBuilder, docRef);
+    verifyDefault();
+    List<CellAttribute> cellAttrList = attrBuilder.build();
+    assertEquals(1, cellAttrList.size());
+    assertEquals("disabled", cellAttrList.get(0).getName());
+  }
+
+  @Test
+  public void test_collectAttributes_addNonEmptyAttribute_value_label() throws Exception {
+    DocumentReference docRef = new DocumentReference("wikiName", "Celements", "TestXClassName");
+    expect(strucDataEdSrvMock.getSelectCellDocRef(docRef)).andReturn(
+        Optional.<DocumentReference>absent());
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_DISABLED))).andReturn(Optional.of(false));
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_VALUE))).andReturn(Optional.<String>of("value"));
+    expect(modelAccessMock.getFieldValue(eq(docRef), eq(
+        OptionTagEditorClass.FIELD_LABEL))).andReturn(Optional.<String>of("label"));
+    replayDefault();

```

Comment:
please be consistent, either add empty lines before replayDefault or don't - but not both mixed in the same class.

I personally prefer an empty line before replayDefault.